src/routes/index.svelte

```diff
@@ -217,10 +232,11 @@
                 <ul>
                     {#each Array.from(history.values()) as item, idx (item.symbol)}
                         <li
-                            class="border border-solid border-gray-600 p-2 h-32
-                            rounded-md">
+                            class="border border-solid border-gray-600 p-2 h-36
+                            rounded-md relative">
                             <button
-                                class="float-right"
+                                class="absolute top-0 right-0 px-0 m-1 bg-red
+                                text-white rounded-md h-4 w-4"
```

Comment:
The 'x' does not sit nicely in the middle of the background and overflows. I think this would look better if there was no 'h-4'.